tests/tensor_ops/test_getitem.py

```diff
@@ -220,3 +241,27 @@ def test_getitem_basic_w_adv_fwdprop():
 )
 def test_getitem_basic_w_adv_bkprop():
     pass
+
+
+@fwdprop_test_factory(
+    mygrad_func=get_item,
+    true_func=get_item,
+    num_arrays=1,
+    index_to_arr_shapes={0: hnp.array_shapes(max_side=4, max_dims=8)},
```

Comment:
Note that running this locally revealed the mygrad bug: https://github.com/rsokl/MyGrad/issues/272

```suggestion
    index_to_arr_shapes={0: hnp.array_shapes(min_side=0, max_side=4, min_dims=0, max_dims=5)},
```